inventory/inv_scripts/inventory_ui.gd

```diff
@@ -0,0 +1,31 @@
+extends Control
+
+@onready var inv: Inv = preload("res://inventory/player_inventory.tres")
+@onready var slots: Array = $NinePatchRect/GridContainer.get_children()
+
+
+var is_open = false
```

Comment:
This is just a boolean so it is not too important, but for future reference, doing `var is_open = false` declares is_open as an inferred type, doing `var is_open := false` explicitly assigns it to a boolean. I am not 100% sure on dynamically typed primitives in godot but using `:=` here would be good practice